### ISSUE TITLE
Feature Request: Twin Snakes or Perfect Balance into 1-2-3 single target during perfect balance

### ISSUE THREAD

**Issue by DanteLuck** [Grammernatzi/XIVComboPlugin#189]
While i've been playing monk for a bit now and testing out this plugin i found that not having access to a 1-2-3 single target made it harder to get the solar and lunar combo to go off, so far i've been playing with the following options enabled

Leaping opo combo
Dragon kick combo
perfect balance to demolish
meditation reminder
AoE Combo
- Form shift option
- meditation feature

draconic leap
twin raptor
demolishing pounce
dragon kick/leaping opo balance feature
perfect balance feature
riddle of fire to brotherhood
riddle of fire to riddle of wind

while having access to shadow of the destroyer + 1-2-3 aoe combo during perfect balance is great for getting both solar and lunar i would also like to have an option akin to this but for single.

Because right now my dragon kick/ leaping opo keeps alternating between eachother during perfect balance and demolish and pouncing coeurl for the perfect balance button itself and twin snakes just sits there as a filler button just for perfect balance moments. 

Maybe i've missed something but if not, would it be possible to turn either **Twin Snakes** or **Perfect Balance**(would be most optimal) into the 1-2-3 combo during Perfect balance as an alternative option. (then it continues on the main combo if needed) so that we can get the solar one without more buttons clogging up the hotbar.

**Comment by Grammernatzi** [comment]
I apologize but I already decided a while ago that I wouldn't do any automatic Perfect Balance solving on the single Target side (unless MKhayle added it or whatever). Currently I have 1 bootshine, 2 dragon kick, 3 form shift and 4 perfect balance (I use the perfect balance fury feature as I feel making it auto for the combos was a mistake, but, eh, too late to remove it). With form shift to twin snakes, this already means you don't need any more buttons. i main MNK atm and it plays very well like this.

**Comment by DanteLuck** [comment]
I see, right now im using 
1: Aoe Combo - placing out masterful blitz
2: opo combo (naturally goes through the whole thing)
ctrl + 2: twin snakes, for only to be used with perfect balance to get the rising phoenix combo
3: perfect balance which turns into demolish + pouncing coeurl
4: form shift which turns into the aoe combo during perfect balance

so i would hit 3 for perfect balance, then 1-1-1 for elixir, then 3 for PB and 4-4-4 for rising phoenix
for single it'd be 3-ctrl+2-2 which mind you isnt streneous at all, was more or less curious if there could be an option to set PB or twin snakes to replace the 1-2-3, das all.
I've got to say though, before this iteration of XIVCombo i didnt even touch monk, i'm having a blast with it, i've tried playing dragoon too and my god is it very streneous to level 1-70, monk was more or less a breeze and i had a really good time doing it, so regardless, 
Thanks for making the mod 😄

**Comment by Grammernatzi** [comment]
Well, I can't account for everyone's hot bars. They made a lot of changes so I simply tried to make it work and it's quite comfortable now. I would absolutely advise people to try changing their hot bar because yes it messes with muscle memory but they changed things so much that it's kind of necessary now to feel smooth unless you're playing vanilla. I had to do it too. My hotbar is nothing like what it was during Endwalker atm. That's just kind of the price of using something like this.
